crates/torii/types-test/abis/base/world.json

```diff
@@ -0,0 +1,657 @@
+[
```

Comment:
I think we can avoid outputting this abi too, since it is standardized. we will provide interfaces for this in the sdks

Reply:
Yep we should be good removing this one too.
The bindgen for `dojo-world` is done via `cargo run --bin dojo-world-abigen`, which uses the `dojo_world.contract_class.json` outputted by scarb and not the manifest.

Reply:
okay so abi's are only needed for user defined `DojoContract`s (i.e. systems), correct?

Reply:
yes that is correct